taxonomies/ingredients.txt

```diff
@@ -13499,6 +13499,19 @@ de:Salz aus der Camargue
 fr:Sel de camargue
 nl:Zout uit de Camargue
 
+<en:sea salt
+en:sel gris, grey salt
```

Comment:
```suggestion
en:grey salt, sel gris
```

Are you sure "gray salt" is always sea salt?

Reply:
"Sel gris" is the real term in english (https://en.wikipedia.org/wiki/Sel_gris), grey salt is a term I've found on the wikipedia and wikidata page. I'm ok with deleting "grey salt" though.

Reply:
> Sel gris (pl. sels gris, "gray salt" in French) is a coarse granular sea salt

From Wikipedia

Reply:
It seems it's always sea salt: https://oureverydaylife.com/what-is-gray-salt-12234109.html

Reply:
We should put the term that is most common on ingredients list. I have not found "sel gris" on any English ingredient list. But we do have some instances of "grey salt".

grey-salt-from-guerande | 3 | *
-- | -- | --
island-of-re-grey-sea-salt | 1 | *
grey-sea-salt | 1 | *
fr:thefine-table-salt-is-made-from-dried-grey-salt | 1 | *
fr:french-grey-sea-salt | 1 | *
fr:coarse-grey-salt-is-harvested-every-day-in-summerfrom-the-clay-beds-of-the-salt-marshes | 1 | *

Wikipedia is not always right. :)